Submodels/MTSOL_call.py

```
@@ -306,78 +381,76 @@ def SetViscous(self,
+            elif line.startswith(' V1,2..') and completion_type == CompletionType.PARAM_CHANGE:
+                return ExitFlag.COMPLETED
             
             # If the solver crashes, return the crash exit flag
             elif line == "" and self.process.poll() is not None:
-                exit_flag = ExitFlag.CRASH.value    
-                break 
+                return ExitFlag.CRASH    
+            
+            # We can also check if the solver will crash by checking for neg. temp. lines in the console output:
+            elif line.startswith(' *** Neg. temp.'):
+                return ExitFlag.CRASH
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)